tracekit.js

```diff
@@ -183,13 +198,13 @@ TraceKit.report = (function reportModuleWrapper() {
      */
     function traceKitWindowOnError(message, url, lineNo, columnNo, errorObj) {
         var stack = null;
-
+      TraceKit.windowPointer = this;
```

Comment:
So this modifies the pointer which might of been set earlier when you subscribed. I'm not sure this should be updated as it would change the subscription handler? Thoughts?